CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.18.14 (2020-11-04)
+
+### Improvements
+
+- **Dialog** Ability to define data to inject be injected into dialog component
```

Comment:
looks like there is a typo in here, I'm not able to find the proper sentence though